modules/tinymce/changelog.txt

```diff
@@ -32,6 +32,7 @@ Version 5.6.0 (TBD)
     Fixed some minor memory leaks that prevented garbage collection for editor instances #TINY-6570
     Fixed resizing a `responsive` table not working when using the column resize handles #TINY-6601
     Fixed an issue where spaces were not preserved in pre-blocks when getting text content #TINY-6448
+    Fixed a regression that caused the selection to be hidden in tables with backgrounds #TINY-6495
```

Comment:
Maybe "difficult to see" or "difficult to differentiate"?

Reply:
Remember this needs to be from the customers perspective. Before you couldn't see it, so from their perspective it was hidden.

Reply:
Decided to go with `difficult to see` given it can be seen on some screens.